src/client/public/scripts/dash/dash.ts

```diff
@@ -426,9 +431,20 @@ async function start() {
     }
   });
 
-  // This allows you to see another users dashboard.
-  const queryParams = new URLSearchParams(window.location.search);
-  startPolling(queryParams.get('login'));
+  if (navigator.serviceWorker) {
+    navigator.serviceWorker.addEventListener('message', (event) => {
+      if (!event.data || !event.data.action) {
+        return;
+      }
+
+      const swMessage = event.data;
+      if (swMessage.action === 'push-received') {
+        performLongPoll(userLogin);
```

Comment:
(optional) `performLongPoll` is the fn which performs the dashboard update right? Don't really like the name, `updateDashboard()` would make more intuitive sense.